app/helpers/markets_helper.rb

```diff
@@ -0,0 +1,2 @@
+module MarketsHelper
+end
```

Comment:
You might consider deleting generated files like helpers and coffeescript if you aren't using them.